src/objectTranslations/Contact-es.objectTranslation

```diff
@@ -3471,66 +3471,38 @@
         <help>¿Cuál teléfono debe ser el usado para la mayoría de las comunicaciones con este contacto?</help>
         <label>Teléfono elegido</label>
         <name>PreferredPhone__c</name>
-        <picklistValues>
-            <masterLabel>Home</masterLabel>
-            <translation>Teléfono particular</translation>
-        </picklistValues>
```

Comment:
@lifewithryan @robertbwatson looks like translations for old picklist values are completely gone.